tutorials-v4/python-introduction/NumPy_Array_Basics.md

```diff
@@ -0,0 +1,320 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+```python
+import numpy as np
+```
+
+# Introduction to NumPy Arrays
+
+
+J.R. Johansson and P.D. Nation
+
+For more information about QuTiP see [http://qutip.org](http://qutip.org)
+
+
+## Introduction
+
+<!-- #region -->
+Until now we have been using lists as a way of storing multiple elements together.  However, when doing numerical computations, lists are not very good.  For example, what if I wanted to add one to a list of numbers?  We couldn't write
+
+```python
+a=[1,2,3]
+a=a+1
```

Comment:
I expected these lines to have been formatted to match flake8, but somehow they are not. Do you know why they weren't?

Reply:
As you mentioned in the comment above, this is probably due to the `#region` tags. I saw these when I used PyCharm to edit Jupyter Notebooks. Maybe someone did the same thing before..
But I don't know why `black` and `flake8` seem to skip this section.